studio-android/LightNovelLibrary/app/src/main/java/org/mewx/wenku8/global/api/VolumeList.java

```diff
@@ -12,4 +19,28 @@
     public int vid;
     public boolean inLocal = false;
     public ArrayList<ChapterInfo> chapterList;
```

Comment:
Please add an empty line between variable definitions and method implementations